egs/wsj/s5/steps/nnet/make_denlats.sh

```diff
@@ -155,7 +155,7 @@ if [ $sub_split -eq 1 ]; then
   for n in `seq $nj`; do
     [ ! -d $dir/lat$n ] && mkdir $dir/lat$n;
     cat $sdata/$n/feats.scp | \
-    awk -v dir=$dir -v n=$n '{ utt=$1; utt_noslash=gensub("/","_","g",utt);
+    awk -v dir=$dir -v n=$n '{ utt=$1; utt_noslash=utt; gsub("/","_",utt_noslash);
```

Comment:
did you test these on a mac?

Reply:
I tested this syntax on my ubuntu 16.04.